replica/tablets.cc

```diff
@@ -252,15 +252,11 @@ future<tablet_metadata> read_tablet_metadata(cql3::query_processor& qp) {
             for (auto&& r : tablet_replicas) {
                 pending.erase(r);
             }
-            if (pending.empty() && transition != tablet_transition_kind::rf_change) {
-                throw std::runtime_error(format("Stage set but no pending replica for table {} tablet {}",
-                                                table, current->tid));
-            }
+            std::optional<tablet_replica> pending_replica;
```

Comment:
why the reordering here?
should it be squashed into the original patch?

Reply:
Removed the reordering.